scipy/special/_precompute/expn_asy.py

```diff
@@ -10,16 +10,11 @@
 from __future__ import division, print_function, absolute_import
 
 import os
-import warnings
 
 try:
-    # Can remove when sympy #11255 is resolved; see
-    # https://github.com/sympy/sympy/issues/11255
-    with warnings.catch_warnings():
```

Comment:
Note that this code will only get run in the test suite if SymPy >= 1.0 is installed, in which case I think you would still get warnings here?